ci/run_cugraph_service_pytests.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
-# Copyright (c) 2024, NVIDIA CORPORATION.
+# Copyright (c) 2024-2025, NVIDIA CORPORATION.
 
 set -euo pipefail
 
 # Support invoking run_cugraph_service_pytests.sh outside the script directory
 cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../python/cugraph-service
 
-pytest --capture=no --cache-clear --benchmark-disable -k "not mg" "$@" tests
+pytest --capture=no --cache-clear --benchmark-disable -m "not mg" "$@" tests
```

Comment:
Maybe this intends to filter test names and not the `mg` mark? If so, we probably aren't using the mark correctly.
```suggestion
pytest --capture=no --cache-clear --benchmark-disable -k "not mg" "$@" tests
```